language/move-prover/tests/sources/defines.exp

```diff
@@ -0,0 +1 @@
+All good, no errors!
```

Comment:
At the top of these .exp file, I see either 

returns: exiting with boogie verification errors

or 

All good, no errors!

Is "returns" a keyword indicating something returned by the tool.  If that is correct, shouldn't it be:

returns: All good, no errors!

Reply:
The returns is the tool Result output. I can change this to make it clearer.